src/SDKs/ApplicationInsights/ApplicationInsights.sln

```diff
@@ -41,6 +45,30 @@ Global
 		{4A1E2ACF-218C-481F-A501-7BB3F3C0B873}.Portable-Release|Any CPU.Build.0 = Release|Any CPU
 		{4A1E2ACF-218C-481F-A501-7BB3F3C0B873}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{4A1E2ACF-218C-481F-A501-7BB3F3C0B873}.Release|Any CPU.Build.0 = Release|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Net45-Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Net45-Debug|Any CPU.Build.0 = Debug|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Net45-Release|Any CPU.ActiveCfg = Release|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Net45-Release|Any CPU.Build.0 = Release|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Portable-Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{BC9AA77F-1605-4386-B04E-368B1363F395}.Portable-Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
What is the intention behind building against portable configurations

Reply:
None -- not sure when this snuck in. I removed it while changing up the directory structure.